strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/services/ArtifactManagementServiceImplTest.java

```diff
@@ -679,6 +688,50 @@ public void testChecksumsStorage(@MavenRepository(repositoryId = "checksums-stor
         assertThat(actualChecksums).isEqualTo(expectedChecksums);
     }
 
+    @ExtendWith({RepositoryManagementTestExecutionListener.class,
```

Comment:
Please, reformat the code below according to our coding convetion. Thanks!

Reply:
Updated to follow coding convention.